clients/logger/logger.go

```diff
@@ -249,8 +251,15 @@ func (lc edgeXLogger) buildLogEntry(logLevel string, msg string, args ...interfa
 
 // Send the log as an http request
 func (lc edgeXLogger) sendLog(logEntry models.LogEntry) {
+	// create a new URL client with an empty URL to fulfil the PostJsonRequest contract while relying wholly on
+	// logTarget for the actual endpoint data
+	emptyURLClient := urlclient.New(types.EndpointParams{
+		UseRegistry: false,
+		Url:         "",
+	}, nil)
+
```

Comment:
Don't do this.  See my comment on this PR that deals with similar behavior implemented elsewhere.